src/utils/utils.ts

```diff
@@ -455,7 +467,7 @@ export function createPostgresPool(
             ? {
                   rejectUnauthorized: false,
               }
-            : undefined,
+            : ssl,
```

Comment:
Maybe we should support these env vars on Heroku, too? Not familiar with why Heroku is setup this way, @Twixes ?

Reply:
That should be good if `ssl` also has `rejectUnauthorized: false` then. Though I was under the impression that an SSL setup with the certs correctly specified shouldn't need `rejectUnauthorized: false`. Unfortunately on Heroku Postgres (for which `process.env.DYNO` is a proxy) we don't know the certs due to the way these DBs are provisioned, but someone deliberately configuring this would be safer with `rejectUnauthorized: true` – as far as I know, I may be wrong.

Reply:
Ah, this makes perfect sense! @mrooding I'm guessing this would work for you as well? i.e. setting `rejectUnauthorized` to `true` when users are providing their own certs and keys.

Reply:
`rejectUnauthorized` should indeed be true. Skipping verification should definitely not be the default. Maybe it would be good to allow the user to set this using an environment variable. If anyone's using self-signed certs they wouldn't be able to get the connection working

Reply:
@neilkakkar I added a new config property for the `rejectUnauthorized` which defaults to true. @Twixes I left the heroku setting hardcoded to `true` since you mentioned that Heroku manages its own certs and I'm assuming that it's impossible to define your own self-signed certs on Heroku.